src/components/data-management/UploadDetailsModal.jsx

```diff
@@ -37,6 +37,7 @@ const UploadDetailsModal = (props) => {
   }, [replacementFileBundle]);
 
   const isSuccessModal = status === UploadStatus.UPLOADED;
+  const isNotUploaded = status === UploadStatus.FILE_NOT_FOUND;
```

Comment:
could it be isNotUploadedModal instead? to be more coherent with the uploaded status check in line 39